src/gui/abstractfileitemwidget.cpp

```diff
@@ -177,18 +183,22 @@ void AbstractFileItemWidget::setFileTypeIcon(const QString &ressourcePath) {
 
 void AbstractFileItemWidget::setFileName(const QString &path, NodeType type) {
     setFileTypeIcon(CommonUtility::getFileIconPathFromFileName(path, type));
-    QString test = QFileInfo(path).fileName();
     _filenameLabel->setText(QFileInfo(path).fileName());
 }
 
 void AbstractFileItemWidget::setPath(const QString &path) {
+    QColor color = GuiUtility::isDarkTheme() ? folderIconColorDark : folderIconColorLight;
     _driveIconLabel->setPixmap(
-        KDC::GuiUtility::getIconWithColor(":/client/resources/icons/actions/icon-folder-empty.svg").pixmap(iconSize));
+        KDC::GuiUtility::getIconWithColor(":/client/resources/icons/actions/icon-folder-empty.svg", color).pixmap(iconSize));
 
-    QString printablePath = "/" + path;
+    QFileInfo fInfo(path);
+    QString printablePath;
+    if (!fInfo.isAbsolute()) printablePath = "/";
+    printablePath += path;
```

Comment:
I believe there are multiple places in the code where we build a `printablePath` (look for the string `+ ...`). It would be nice to have helper function that would handle them all.